### PR TITLE
Add more properties

### PR DESCRIPTION
**Pull request by Klrfl**
I have added some more properties. Please review them first, thank you!!

### CODE REVIEW

File: properties.js [hunk 1]
```diff
@@ -1,102 +1,289 @@
...
   'animation-direction': 'arah-animasi',
   'animation-duration': 'durasi-animasi',
+  'animation-fill-mode': 'mode-isi-animasi',
   'animation-iteration-count': 'berapa-kali-animasi-dijalankan',
   'animation-name': 'nama-animasi',
-  'animation': 'animasi',
+  'animation-timing-function': 'fungsi-waktu-animasi',
+  
+  'appearance': 'penampakan',
+  'aspect-ratio': 'rasio-aspek',
```
Comment (karsanda) [inline properties.js:19]: this should be "aspek-rasio"

File: values.js [hunk 2]
```diff
@@ -1,25 +1,155 @@
 const values = {
-  'absolute': 'absolut',
+  // global values
+  'inherit': 'warisan',
+  'initial': 'awal',
+  'revert': 'balikkan',
```
Comment (karsanda) [inline values.js]: I think "kembalikan" is better

File: values.js [hunk 3]
```diff
@@ -1,25 +1,155 @@
...
+  'revert-layer': 'balikkan-lapisan',
```
Comment (karsanda) [inline values.js]: same as above "kembalikan-lapisan" is better I think

**Review by karsanda — DISMISSED** [review]
Hi! this is great! I've added some comments and suggestions for the words used.

Thanks bro!

File: values.js (hunk 2 above)
Comment (Klrfl) [reply values.js]: If so, then 'warisan' should be 'wariskan' right? I'll change it.

File: values.js (hunk 3 above)
Comment (Klrfl) [reply values.js]: I agree. I'll change

**Comment by Klrfl** [comment]
I have made the changes you requested. Please review!

**Review by karsanda — APPROVED** [review]
lgtm

**Pull request merged**